tests/unit/test_get_movie_by_title.py

```diff
@@ -1 +1,27 @@
-# TODO: Feature 3
+# TODO: Feature 3
+
+from src.models.movie import Movie
+from src.repositories.movie_repository import get_movie_repository
+from app import app
+import pytest
+movie_repository = get_movie_repository()
+
+#test what happens if the move is not in the list
+def test_search_movies_No_Movie():
+    test_app = app.test_client()
+    #this gets the page
+    response = test_app.get('/movies/search?title=dad')
+    assert b' <h2>Enter in a valid movie</h2>' in response.data
+
+#test what happens if the movie is in the list
+def test_search_movies_yes_Movie():
+    #this should make the movie
+    movie = Movie('YoMama', 'George Lucas', 5)

```

Comment:
instead of movie = Movie(...) are you trying to use the create_movie() function?